translations/difficulty.tex/pl.po

```diff
@@ -144,9 +144,9 @@ msgid ""
 "instead.} \\\\ \\end{tabularx} }"
 msgstr ""
 "\\hommtable{38}{ \\centering \\textbf{Tabela zasad opcjonalnych}\\\\ \\small "
-"{ \\color{white} Dzięki tym dodatkowym zasadom możesz urozmaicić swoje "
-"rozgrywki i modyfikować ich trudność.\\\\ } \\bigskip \\begin{tabularx}"
-"{\\linewidth}{>{\\hsize=0.3\\hsize\\linewidth=\\hsize}X "
+"{ \\color{white} Dodatkowymi zasadami możesz urozmaicić swoje rozgrywki i "
```

Comment:
Will `Dzięki dodatkowym zasadom` do? The suggested change feels less natural

Reply:
That will do. It still fits. Thanks!